imageregistry/v1/01-crd.yaml

```
@@ -617,6 +617,10 @@ spec:
                 pruner jobs to retain. Defaults to 3 if not set.
               type: integer
               format: int32
+            ignoreInvalidRefs:
```

```suggestion
            ignoreInvalidImageReferences:
```

makes it more explicit in case you need to ignore other types of references in the future for some reason.

---

Renamed. I couldn't choose between two options: name it the same as the flag (--ignore-invalid-refs), or give it а more natural name.

---

we can always deprecate/rename the cli arg to get things consistent in the future, probably better to move the api towards what we consider the better name to be (assuming we agree it's better).